umbral-pre-python/Cargo.toml

```diff
@@ -8,6 +8,6 @@ edition = "2018"
 crate-type = ["cdylib"]
 
 [dependencies]
-pyo3 = { version = "0.12.3", features = ["extension-module"] }
+pyo3 = { version = "0.13", features = ["extension-module"] }
```

Comment:
`pyo3` dropped support for Python 3.5 in `0.13`, so the `build-wheels.sh` will break

Reply:
Thanks for noticing, I'll correct that.

It's still experimental, I'm not sure how it's supposed to be run - in CI perhaps?

Reply:
Sure, CI is good. I think it could be a bit easier to run on CI actually - The script seems trivially parallelizable and we can provide it with any kind of Python version there.

Reply:
Don't want to rebase the whole thing, fixed in #47